src/convertToObject.js

```diff
@@ -6,7 +6,57 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString
+    .split(';')
+    .map((str) => str.trim())
+    .filter(Boolean);
+
+  const convertedToObject = {};
+
+  splittedString.forEach((str) => {
+    const styleArr = str.split(':');
```

Comment:
The `split(':')` method will not handle cases where there are multiple colons within a style value. Although handling multiple colons is not required, be aware that this could lead to unexpected behavior if such cases are encountered.